packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -114,15 +146,32 @@ export default class QuanticNumericFacet extends LightningElement {
     return I18nUtils.format(label, this.label);
   }
 
+  get start() {
+    return this.isSelected ? '' : this.numericFilter?.state?.range?.start;
```

Comment:
IMO you shouldn't peek in the numeric filter state like that. LWC is not tracking `this.numericFilter` nor any of its children. I think you should instead get the values you need through the `updateState` callback and put them into an object that is tracked by LWC.